packages/core/src/middleware/koa-spa-proxy.ts

```diff
@@ -43,6 +55,14 @@ export default function koaSpaProxy<StateT, ContextT extends IRouterParamContext
       return next();
     }
 
+    const { signInExperiences } = queries ?? {};
```

Comment:
How about setting the`queries` input parameter mandatory, and using the `packagePath === 'experience' to guard the cusomUi logics?

Reply:
updated